### ISSUE TITLE
Failed installation  IJulia

### ISSUE THREAD

**Issue by Marianagg01** [JuliaLang/IJulia.jl#949]
I use Julia 1.5.1 and Windows 10.  When I try to install IJulia by different methods it just sends a message that it is installing, but it´s been 3 hours and there is no change.
![image](https://user-images.githubusercontent.com/67020336/92419647-65fe9d00-f134-11ea-9827-dabedb0c798f.png)
![image](https://user-images.githubusercontent.com/67020336/92419656-731b8c00-f134-11ea-928d-0fea29b934ba.png)

**Comment by nilshg** [comment]
Can be closed.

**Comment by JamesWrigley** [comment]
Yeah I've never been able to reproduce this. But feel free to reply here if it still happens @Marianagg01 and we can reopen it.
